packages/amazonq/src/lsp/chat/commands.ts

```diff
@@ -4,26 +4,30 @@
  */
 
 import { Commands, globals } from 'aws-core-vscode/shared'
-import { window } from 'vscode'
+// import { window } from 'vscode'
 import { AmazonQChatViewProvider } from './webviewProvider'
 
+/**
+ * TODO: Re-enable these once we can figure out which path they're going to live in
+ * In hybrid chat mode they were being registered twice causing a registration error
+ */
```

Comment:
This is will still have to be done as a follow-up right?

Reply:
For some reason these events (send to prompt, etc) seem to work using the current events that get registered. I think we just need to re-visit it once we deprecate codewhisperer chat/the agents that live in VSCode